src/app/(chat)/chat/[chat_id]/components/chat.tsx

```
@@ -0,0 +1,163 @@
+            });
+            console.log("AI message saved (chat page):", aiMessage);
+          } catch (error) {
+            console.error("Error saving AI message (chat page):", error);
+          }
+        }
+      }
+    });
+
+    // Update useChat messages when conversation data changes (e.g., after initial fetch)
+    useEffect(() => {
+      if (conversation && conversation.messages && aiChatMessages.length !== conversation.messages.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)